app/utils/api-errors.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Check if the status error occurs in the response errors collection
+ *
+ */
+export default function hasErrorWithStatus(errorResponse, status) {
+  if (!errorResponse) { return false; }
+
+  return !!errorResponse.errors.find(error => (error.status == status));
```

Comment:
It'd be more efficient to use [isAny](https://api.emberjs.com/ember/3.10/classes/EmberArray/methods/isAny?anchor=isAny) here:
```js
return errorResponse.errors.isAny('status', status));
```